etc/kayobe/environments/ci-aio/reboot.yml

```diff
@@ -0,0 +1,3 @@
+---
+# Ensure that the reboot playbook is always executed using the boostrap user
+reboot_with_bootstrap_user: true
```

Comment:
Can it not be applied at the import_playbook level?

Reply:
I like the idea, I'll try it out